src/sass/components/_scroll-vertical.scss

```diff
@@ -0,0 +1,9 @@
+%scroll-vertical{
+    overflow-y: scroll;
+    scrollbar-width: none;
+
+    &::-webkit-scrollbar {
+        display: none;
+    }
+
+}
```

Comment:
if `scroll-vertical` and `scroll-horizontal` are only placeholders, can it be kept with other placeholders? do they need separate files?

Reply:
Sorry for that. I think it's not need. i will correct it.